src/styles/main-header/main-header.stories.js

```diff
@@ -1,4 +1,4 @@
-import StaticHTML from './header.html'
+import StaticHTML from './main-header.html'
 
 export default {
   title: 'header',
```

Comment:
I'd also rename it here, as this is how it will be shown in storybook.
```suggestion
  title: 'main-header',
```